docs/custom-flows/error-handling.mdx

```diff
@@ -114,11 +114,11 @@ If you have [Account Lockout](/docs/security/user-lock-guide) enabled on your in
       "long_message": "Your account is locked. You will be able to try again in 30 minutes. For more information, please contact support.",
       "code": "user_locked",
       "meta": {
-        "duration_in_seconds": 1800
+        "lockout_expires_in_seconds": 1800
       }
     }
   ]
 }
 ```
 
-The remaining duration will vary [depending on your configuration](/docs/security/customize-user-lockout). You can opt to render the error message returned as-is or format the supplied `duration_in_seconds` as per your liking.
\ No newline at end of file
+The remaining duration will vary [depending on your configuration](/docs/security/customize-user-lockout). You can opt to render the error message returned as-is or format the supplied `lockout_expires_in_seconds` value as per your liking in your own custom error message.
```

Comment:
I still don't know what this means... Can you give example code? Why is this something that is set to a numeric value and not pulled from the settings they set in the dashboard? Right now it looks like you're telling it a different number from whatever is set in the Clerk Dash

Reply:
I re-did this section to make it more evident that `lockout_expires_in_seconds` represents the live value of how many seconds remain until the lockout expires.

Also added a code sample of how to use this value to format a timestamp to one's liking.

Reply:
Nice, thank you!